packages/website/src/widgets/header/header.test.tsx

```diff
@@ -0,0 +1,20 @@
+import '@testing-library/jest-dom'
+import { render, screen } from '@testing-library/react'
+import Header from './header'
+
+describe('Header', () => {
```

Comment:
Include the coverage results in your PR description, like the following example:

---

### Testing

- Ran testing suites:

```
Test Suites: 1 passed, 1 total
Tests:       1 passed, 1 total
Snapshots:   0 total
Time:        3.055 s
Ran all test suites.

- Write new tests:

```
File                                           | % Stmts | % Branch | % Funcs | % Lines | Uncovered Line #s 
-----------------------------------------------|---------|----------|---------|---------|-------------------
website/src/widgets/header                    |     100 |      100 |     100 |     100 |                   
  header.tsx                                   |     100 |      100 |     100 |     100 |